scripts/run.sh

```diff
@@ -9,7 +9,7 @@
 # $ cd /path/to/many-ledger
 # $ ./script/run.sh
 set -pube # .. snicker ..
-TM_MIN_VERSION=0.35.0 # The minimum tendermint version supported
+TM_MIN_VERSION=0.34.0 # The minimum tendermint version supported
```

Comment:
nit: we should add a check that errors if we use TM 0.35

Reply:
This is not overridable by external env variables, so...

Reply:
Done.